bigframes/functions/remote_function.py

```diff
@@ -278,26 +289,45 @@ def generate_cloud_function_code(
 
         # main.py
         entry_point = bigframes.functions.remote_function_template.generate_cloud_function_main_code(
-            def_, directory, is_row_processor
+            def_,
+            directory,
+            input_types=input_types,
+            output_type=output_type,
+            is_row_processor=is_row_processor,
         )
         return entry_point
 
     def create_cloud_function(
         self,
         def_,
         cf_name,
+        *,
+        input_types: Tuple[str],
```

Comment:
Do `input_types` and `output_types` need default values, when they are defined after "*"?

Reply:
No, the `*` means they are keyword-only. Without a default value they are required.